### PR TITLE
Updated Clare's research description, linked website

### PR DESCRIPTION
**Pull request by cmlohrmann**
Spelled name correctly on research page, linked website there, edited content on algoHRI page, also linked website and added new picture

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,4 @@
 source "https://rubygems.org"
 gem 'github-pages'
+
+gem "webrick", "~> 1.8"
```
Comment (alecive) [inline Gemfile:4]: why this?

File: _posts/research/subteams/2021-03-16-algorithmic-social-hri.md [hunk 2]
```diff
@@ -46,19 +46,15 @@ traits of their partner.  In this stream of research, we explore how to imbue au
 </div>
 
 
-## 1.2 Reference Dependent Risk Attitudes in HRI
+## 1.2 Predictability in HRI
```
Comment (alecive) [inline _posts/research/subteams/2021-03-16-algorithmic-social-hri.md:49]: Predictability is good , but "in HRI" is too generic. Pleae make it more compelling!

File: _posts/research/subteams/2021-03-16-algorithmic-social-hri.md [hunk 3]
```diff
@@ -46,19 +46,15 @@ traits of their partner.  In this stream of research, we explore how to imbue au
...
 
-{% include image.html url="research/algo-social-hri/prospect-theory.png" max-width="50%" description="Humans have a tendency to underestimate the value of outcomes." %}
+Our methods leverage the innate human propensity for pattern recognition and abstraction in order to improve team dynamics in human-robot teams and to make robots more predictable to the humans that work with them. Patterns are a cognitive tool that humans use and rely on often, and the human brain is in many ways primed for pattern recognition and usage. In this research stream we lean into human cognitive tendencies to improve human-robot teaming and human perceptions of their robot teammates.
+
+{% include image.html url="research/algo-social-hri/maria_setup_img_blur.png" max-width="50%" description="The setup for a work currently under review, where participants played a coordination game with a Sawyer robot." %}
```
Comment (alecive) [inline _posts/research/subteams/2021-03-16-algorithmic-social-hri.md:57]: Great paragraphs! I would further expand on it and make it more concrete. What it is you did? This is not about the technical part, is about talking about what this intuition brings.

**Review by alecive — CHANGES_REQUESTED** [review]

**Comment by alecive** [comment]
Ref #202

**Comment by alecive** [comment]
@cmlohrmann please update this and close it ASAP!

File: Gemfile (hunk 1 above)
Comment (cmlohrmann) [reply Gemfile]: had a versioning issue locally

File: _posts/research/subteams/2021-03-16-algorithmic-social-hri.md (hunk 2 above)
Comment (cmlohrmann) [reply _posts/research/subteams/2021-03-16-algorithmic-social-hri.md]: reworked title

File: _posts/research/subteams/2021-03-16-algorithmic-social-hri.md (hunk 3 above)
Comment (cmlohrmann) [reply _posts/research/subteams/2021-03-16-algorithmic-social-hri.md:56]: Reorganized and updated with additional images as well

**Comment by cmlohrmann** [comment]
Changes made as asked

File: Gemfile [hunk 4]
```diff
@@ -1,4 +1,2 @@
 source "https://rubygems.org"
-gem 'github-pages'
-
-gem "webrick", "~> 1.8"
+gem 'github-pages'
```
Comment (alecive) [inline Gemfile:2]: What is this change for?

File: _data/people.yml [hunk 5]
```diff
@@ -72,6 +72,7 @@
 
  - name: Clare Lohrmann
    role: phd
+   url: https://cmlohrmann.github.io/
```
Comment (alecive) [inline _data/people.yml:75]: 🎉

**Review by alecive — CHANGES_REQUESTED** [review]

**Comment by alecive** [comment]
@cmlohrmann apart from the minor comment above, you have a conflict because the version of the document you were editing is now updated. Either resolve the conflict or open a new PR!

**Comment by cmlohrmann** [comment]
Updates are now in newest pull request

**Pull request closed without merging**